pkg/build/api/types.go

```diff
@@ -356,6 +356,12 @@ type ImageChangeTrigger struct {
 	// LastTriggeredImageID is used internally by the ImageChangeController to save last
 	// used image ID for build
 	LastTriggeredImageID string
+
+	// From is a reference to an ImageStreamTag that will trigger a build when updated
+	// It is optional. If no From is specified, the From image from the build strategy
+	// will be used. Only one ImageChangeTrigger with an empty From reference is allowed in
+	// a build configuration.
+	From *kapi.ObjectReference
```

Comment:
At this point just @ me - we'll appoint another dictator later.

On Tue, Aug 4, 2015 at 1:29 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/build/api/types.go
> <https://github.com/openshift/origin/pull/3748#discussion_r36216471>:
>
> > @@ -356,6 +356,12 @@ type ImageChangeTrigger struct {
> >  	// LastTriggeredImageID is used internally by the ImageChangeController to save last
> >  	// used image ID for build
> >  	LastTriggeredImageID string
> > +
> > +	// From is a reference to an ImageStreamTag that will trigger a build when updated
> > +	// It is optional. If no From is specified, the From image from the build strategy
> > +	// will be used. Only one ImageChangeTrigger with an empty From reference is allowed in
> > +	// a build configuration.
> > +	From *kapi.ObjectReference
>
> @smarterclayton <https://github.com/smarterclayton> who's on the short
> list for api review?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3748/files#r36216471>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift